src/main/java/ch/uzh/ifi/hase/soprafs21/rest/dto/userDTO/UserPutDTO.java

```diff
@@ -0,0 +1,53 @@
+package ch.uzh.ifi.hase.soprafs21.rest.dto.userDTO;
+
+import ch.uzh.ifi.hase.soprafs21.constant.Gender;
+import ch.uzh.ifi.hase.soprafs21.entities.UserInterests;
+
+public class UserPutDTO {
+
+    private String email;
```

Comment:
should the user be able to update his email?

Reply:
Good point, i think they should have the possibility for cases like if they change university (uzh --> eth for example) and then to not be forced to register again, they could just change the email. But this is a point we can discuss on monday 👍